test.disabled/ejabberd_tests/tests/oauth_SUITE.erl

```diff
@@ -313,6 +315,26 @@ provision_token_login(Config) ->
     Result = escalus:wait_for_stanza(Conn),
     VCard = exml_query:subelement(Result, <<"vCard">>).
 
+check_for_supported_sasl_mechanisms(Config) ->
+    Host = ct:get_config({hosts, mim, domain}),
+    Module = mod_auth_token,
+    AliceSpec = escalus_users:get_userspec(Config, alice),
+    ConnSteps = [start_stream,
+                 stream_features,
+                 maybe_use_ssl,
+                 maybe_use_compression],
+    {ok, _, Features} = escalus_connection:start(AliceSpec, ConnSteps),
+    true = lists:member(<<"X-OAUTH">>, proplists:get_value(sasl_mechanisms,
+                                                           Features, [])),
+    dynamic_modules:stop(Host, Module),
+    {ok, _, NewFeatures} = escalus_connection:start(AliceSpec, ConnSteps),
+    false = lists:member(<<"X-OAUTH">>, proplists:get_value(sasl_mechanisms,
+                                                            NewFeatures, [])),
+    AuthOpts = [{ {validity_period, access}, {60, minutes} },
```

Comment:
Options should be extracted via RPC in `init_per_testcase` (what if default config changes but options here won't get an update?) and appended to test `Config`.